### PR TITLE
Avivash/add loading UI

### PR DESCRIPTION
**Pull request by avivash**
# Description

Adding the loading spinner from other webnative templates into this one

## Type of change

- [x] New feature (non-breaking change that adds functionality)

## Screenshots/Screencaps

**Authed:**
https://www.loom.com/share/6c14c862eea0411993ed464505e4dd35

**Not Authed:**
https://www.loom.com/share/2d144eeb03f54cd186a83fe779704792

### CODE REVIEW

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -54,11 +56,16 @@
...
 <div data-theme={$themeStore} class="min-h-screen">
   <Notifications />
-  <SidebarNav>
-    <Header />
-    <div class="px-4">
-      <slot />
-    </div>
-  </SidebarNav>
+
+  {#if $sessionStore.loading && !$page.url.pathname.match(/register|backup|delegate/)}
```
[the remark this answers is not in the document]
Comment (icidasset) [reply src/routes/+layout.svelte:60]: I know you explained it partially in the description, but I was wondering why not take an other approach and set the `loading` state to false whenever those pages load?

[the remark this answers is not in the document]
Comment (avivash) [reply src/routes/+layout.svelte]: So, this is strange, but I ended up removing the route exceptions and everything works fine. I really though the loading spinner was popping up mid-registration before, but i guess not 😅

**Comment by avivash** [comment]
@icidasset i think this is good to go now if you're happy with it 👍🏼 I applied the same change to the WAT-react PR

File: src/lib/init.ts [hunk 2]
```diff
@@ -9,7 +10,7 @@ export const initialize = async (): Promise<void> => {
 
     const program: webnative.Program = await webnative.program({
       tag: { creator: 'Fission', name: 'WAT' },
-      debug: false // TODO: Add a flag or script to turn debugging on/off
+      debug: dev
```
Comment (avivash) [inline src/lib/init.ts:13]: Also, added this to automatically toggle `debug` on in `dev` mode 👍🏼

**Review by bgins — APPROVED** [review]
Looks good! Let bring it in. 💯 🎉

**Pull request merged**